frontend/src/Settings/CustomFormats/CustomFormats/Specifications/EditSpecificationModalContent.js

```diff
@@ -41,7 +41,7 @@ function EditSpecificationModalContent(props) {
   return (
     <ModalContent onModalClose={onCancelPress}>
       <ModalHeader>
-        {`${id ? translate('EditCondition') : translate('AddCondition')} - ${implementationName}`}
+        {id ? translate('EditConditionImplementation', { implementation: implementationName }) : translate('AddConditionImplementation', { implementation: implementationName })}
```

Comment:
I think changing these all to `implementationName` would be good, it'll shorten down every call to translate and is a bit more explicit as to what "implementation" is.

```suggestion
        {id ? translate('EditConditionImplementation', { implementationName }) : translate('AddConditionImplementation', { implementationName })}
```